### DISCUSSION TITLE
personal-development-khordad-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#14] [Announcements]
# personal-development-khordad-1402/

گزارش توسعه مهارتی من در خرداد ۱۴۰۲.

https://aprd.ir/personal-development-khordad-1402/

**Comment by amirghh** [comment]
سلام
برای بازی که هم شانس باشه هم استراتژی فکر کنم تخته نرد باحال باشه
در مورد جابجایی وسایل خونه: منم اسفند پارسال شی چهرشنبه سوری جابجا شدم. کلی تو سایتها و آچاره و... گشتم دنبال شرکت خوب . کامنت ها مختلف بود تا اینکه یکی از دوستام گفت همه اشون یه قیمتی میدن و بعد انتهای کار یه وسیله سنگین مثل یخچال رو نمیارن بالا و تسویه حساب میکنن بعد میارن. اونجا هم نرخشون کاملا متفاوت از چیزی هست که گفتی و اگر هم جر وبحث کنی دوره ات میکنن و...
به من پیشنهاد ظریف بار اصلی رو داد(فقط ظریف باری که سمت تهرانپارس هست و شماره تلفن 1500) گفت فقط اصلی باشه بقیه فیک هستن.
انصافا قیمت رو هرچی طی کرده بودن گرفتن و حتی برای انعام حرفی نزدن. کارشون هم عالی بود
 
املاکی ها رو هیچ کس جز خودشون نحوه برقراری رابطه رو نمیدونه. فقط پول تعیین کننده هست اونم فقط وقتی قرار باشه بره تو جیب اونها. به جرات میگم تابحال کسی رو ندیدم که از دست این قماش راضی باشه. چون یه کار تخصصی افتاده دست عده ایی دلال که میخوان یک شبه پولدار بشن. حتی تو حوزه کاری که میکنن ذره ایی اطلاعات ندارن.

**Reply by pourmand1376** [reply]
سلام آقا امیر. خیلی ممنون از پیشنهادتون. دفعه بعد برای اسباب‌کشی استفاده می‌کنم.

در مورد بنگاه فکر می‌کنم که در آینده نزدیک مجبور بشن که تغییرات مهمی در روند کاریشون بدند؛ چون به این شکل ارزش خاصی تولید نمی‌کنند. این‌طور که شنیدم در بعضی کشورها مانند هلند، کار بنگاه املاک مواردی نظیر هماهنگی تعمیرات خانه رو هم دربرگرفته. در اون صورت شاید بتونند بصورت پایدار کارشون رو ادامه بدند.

**Comment by hesammarhemati** [comment]
سلام امیر جان

امیدوارم که اوضاع احوالت خوب باشه و کارها به خوبی پیش برن

تا یکی دو سال پیش توافقاتی که انجام می‌دادم به صورت شفاهی بود و جز موارد خیلی ضروری و مهم، میل چندانی به نوشتن و مکتوب کردن نداشتم و  به مشکل خاصی هم بر نخورده بودم تا این‌که در مورد یک قرارداد سر ننوشتن صحبت‌هایی که انجام دادیم ضرر قابل توجهی بهم وارد شد و همیشه به دوستم می‌گم که من چوب توافق شفاهی خوردم 
اما بعد از این اتفاق دیگه هر چیزی کوچکی رو می‌نوسیم؛ به عنوان مثال وقتی می‌خوام در مورد هزینه اسباب کشی به 
 توافقی برسم اون رو تو قالب یه پیام یا در قسمت چت مجموعه‌هایی که این خدمات رو ارائه می‌دن به صورت خیلی واضح و شفاف می‌نویسم

مثلاً می‌نویسم که یه یخچال فلان مدل دارم که از طبقه فلان در مبدا  می‌ره به طبقه فلان در مقصد و  در هر دو نقطه و یخچال از طریق راه پله جابجا میشه و در مقصد هم یچخال تا محل قرارگیری اون باید جابجا بشه. و اگه خیلی جزئی‌تر بخوام بنویسم تعداد پاگرد یا تعداد پله‌ها رو ذکر می‌کنم

با یه شرکتی که کار فروش و نصب و راه اندازی آسانسور کار می‌کنم که واقعاً خدمات خوبی ارائه می‌دن و کارشون رو به معنی کلمه درست انجام می‌دن و هر بار که من می‌خوام ازشون آسانسور بخرم یه پوشه جدا درست می‌کنه و از اولین روز مذاکرات و فرستادن پیش فاکتور همه چیز رو مستند می‌کنن؛ مثلا می‌گی ترتیب دکمه‌های کابین این شکلی باشه بعد درجا همون رو رو کاغذ می‌کشه و ازت امضا می‌گیره یا تمام صحبت‌هایی که تو هر جلسه می‌شه رو مستند می‌کنه و زیرش رو با ساعت و تاریخ امضا می‌کنه 

جالب این‌جا بود که یه بار سر یه موضوعی اختلاف نظر داشتیم و به منشیش گفت که پرونده مربوط به اون ساختمون رو بیاره و هزار تا ورق زد تا صحبت‌هایی که اون روز بین‌مون انجام شد رو پیدا کرد و باهم دیگه خوندیم

مستند سازی دقیق و شفاف خیلی راهگشاست امیر جان

**Reply by pourmand1376** [reply]
خیلی ممنون آقا حسام.

نکته خیلی مهمی رو به من گوشزد کردید. قبلاً چند بار از افراد مختلف شنیده بودم ولی یه حرف خوب رو هر چقدر آدم بشنوه باز هم کمه. 

امیدوارم که بتونم در عمل به توصیه‌تون عمل کنم.

**Reply by amirghh** [reply]
سلام
بله دقیقا منم مدتی میشه برای هر کاری که کمی اهمیت برام داره قرارداد حتی در حد همون چت رو پیش بینی میکنم. مثلا در مورد نقاشی خونه روی یک برگ آ4 تمام چیزی رو که میخواستم نوشتم و نقاش هم تمام وکمال انجام داد. وقتی چیزی مکتوب میشه انگار هر دوطرف تعهد بیشتری احساس میکنن تا وقتی شفاهی هست.
